# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
#   <jonny_nut@hotmail.com>, 2012.
# Maxim Dziumanenko <dziumanenko@gmail.com>, 2003.
# Yuri Chornoivan <yurchor@ukr.net>, 2012.
# Yuri Chornoivan <yurchor@ukr.net>, 2015. #zanata
# Yuri Chornoivan <yurchor@ukr.net>, 2016. #zanata
# Yuri Chornoivan <yurchor@ukr.net>, 2018. #zanata
msgid ""
msgstr ""
"Project-Id-Version: Spacewalk\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-04-25 09:41+0200\n"
"PO-Revision-Date: 2018-03-16 12:50+0000\n"
"Last-Translator: Yuri Chornoivan <yurchor@ukr.net>\n"
"Language-Team: Ukrainian <trans-uk@lists.fedoraproject.org>\n"
"Language: uk\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: Zanata 4.6.2\n"

#: ../server/apacheAuth.py:72
#, python-format
msgid "incomplete proxy authentication token: %s"
msgstr "неповний ключ розпізнавання на проксі: %s"

#: ../server/apacheAuth.py:103
#, python-format
msgid "Sent proxy signature %s does not match ours %s."
msgstr "Надісланий ​​підпис проксі %s не збігається з нашим %s."

#: ../server/apacheAuth.py:142
msgid "declined proxy authentication"
msgstr "у розпізнаванні на проксі-сервері відмовлено"

#: ../server/apacheRequest.py:67
#, python-format
msgid "Invalid request received (%s)."
msgstr "Отримано неприпустимий запит (%s)."

#. Send the message as a fault.
#: ../server/apacheRequest.py:126
#, python-format
msgid ""
"IMPORTANT MESSAGE FOLLOWS:\n"
"%s"
msgstr ""
"ДАЛІ ВКАЗАНО ВАЖЛИВЕ ПОВІДОМЛЕННЯ:\n"
"%s"

#: ../server/apacheRequest.py:156
#, python-format
msgid ""
"While running '%s': caught\n"
"%s : %s\n"
msgstr ""
"Під час роботи «%s» отримано\n"
"%s : %s\n"

#. pylint: disable-next=consider-using-f-string
#: ../server/apacheRequest.py:671
#, python-format
msgid "Invalid URI %s"
msgstr "Некоректна адреса %s"

#: ../server/handlers/app/packages.py:220
#, python-format
msgid ""
"\n"
"                   %s Account has been deactivated on this server.\n"
"                   Please contact your Org administrator for more help."
msgstr ""
"\n"
"                   На цьому сервері вимкнено обліковий запис %s.\n"
"                   Щоб дізнатися більше, будь ласка, зверніться до "
"адміністратора вашої організації."

#: ../server/handlers/xmlrpc/proxy.py:71
#, fuzzy, python-format
msgid ""
"SUSE Multi-Linux Manager Proxy service not enabled for server profile: \"%s\""
msgstr "Проксі-сервіс Spacewalk не включений для профілю сервера: «%s»"

#: ../server/handlers/xmlrpc/proxy.py:94
msgid "Invalid session key"
msgstr "Неприпустимий ключ сеансу"

#: ../server/handlers/xmlrpc/proxy.py:102
msgid "This server ID no longer exists"
msgstr "Цього ідентифікатора сервера більше не існує"

#: ../server/importlib/userAuth.py:109
msgid "You are not authorized to manage packages in the null org"
msgstr ""
"У вас немає уповноважень на керування пакунками у організації із порожньою "
"назвою"

#: ../server/importlib/userAuth.py:115
#, python-format
msgid "You are not allowed to manage packages in the %s org"
msgstr "У вас немає прав на керування пакунками у організації %s"

#. ok, you're a regular user who doesn't manage any channels.
#. take a hike.
#: ../server/importlib/userAuth.py:132
msgid "You are not allowed to perform administrative tasks"
msgstr "Ви не маєте права виконувати завдання з адміністрування"

#: ../server/importlib/userAuth.py:159
#, python-format
msgid ""
"You are not allowed to manage channel %s, or that channel does not exist"
msgstr "У вас немає права керувати каналом %s або цього каналу не існує"

#: ../server/importlib/userAuth.py:229
#, fuzzy, python-format
msgid ""
"\n"
"               %s Account has been deactivated on this server.\n"
"               Please contact your Org administrator for more help."
msgstr ""
"\n"
"                   На цьому сервері вимкнено обліковий запис %s.\n"
"                   Щоб дізнатися більше, будь ласка, зверніться до "
"адміністратора вашої організації."

#: ../server/rhnChannel.py:2217
#, python-format
msgid ""
"Your account does not have access to any channels matching "
"(release='%(release)s', arch='%(arch)s')%(www_activation)s"
msgstr ""
"Ваш обліковий запис не має доступу до жодного із відповідних каналів "
"(випуск=«%(release)s», архітектура=«%(arch)s»)%(www_activation)s"

#: ../server/rhnChannel.py:2236
#, python-format
msgid "Insufficient subscription permissions for release (%s, %s"
msgstr "Недостатні права передплати на випуск (%s, %s"

#. Various messages that can be reused
#.
#. bretm 02/07/2007 -- when we have better old-client documentation, probably
#. will be safe to get rid of all this crap
#: ../server/rhnChannel.py:2550 ../server/rhnChannel.py:2569
msgid "System Registered but Inactive"
msgstr "Систему зареєстровано, але не активовано"

#: ../server/rhnChannel.py:2552
#, python-format
msgid ""
"\n"
"Invalid Architecture and OS release combination (%s, %s).\n"
"Your system has been registered, but will not receive updates\n"
"because it could not be subscribed to a base channel.\n"
"Please contact your organization administrator for assistance.\n"
msgstr ""
"\n"
"Некоректна комбінація архітектури і випуску операційної системи (%s, %s).\n"
"Вашу систему зареєстровано, але вона не отримуватиме оновлень, оскільки\n"
"її не вдалося підписати на базовий канал.\n"
"Будь ласка, зв’яжіться із адміністратором вашої організації, щоб отримати "
"допомогу.\n"

#: ../server/rhnChannel.py:2561
#, python-format
msgid ""
"\n"
"  This system has been successfully registered, but is not yet entitled\n"
"  to service.  To entitle this system to service, login to the web site at:\n"
"\n"
"  %(entitlement_url)s\n"
msgstr ""
"\n"
"  Цю систему успішно зареєстровано, але її ще не допущено до\n"
"  обслуговування. Щоб допустити цю систему до обслуговування, увійдіть до "
"системи на цьому сайті:\n"
"\n"
"  %(entitlement_url)s\n"

#: ../server/rhnChannel.py:2571
#, python-format
msgid ""
"\n"
"  This system has been successfully registered, but no service entitlements\n"
"  were available.  To entitle this system to service, login to the web site "
"at:\n"
"\n"
"  %(entitlement_url)s\n"
msgstr ""
"\n"
"  Цю систему успішно зареєстровано, але не виявлено допусків до "
"обслуговування.\n"
"  Щоб допустити цю систему до обслуговування, увійдіть до системи на цьому "
"сайті:\n"
"\n"
"  %(entitlement_url)s\n"

#. Invalid server certificate.
#: ../server/rhnHandler.py:62
msgid "Please run rhn_register as root on this client"
msgstr ""
"Будь ласка, запустіть для цієї клієнтської системи rhn_register від імені "
"адміністратора (root)"

#: ../server/rhnHandler.py:76
#, python-format
msgid "Service not enabled for system profile: \"%s\""
msgstr "Службу не увімкнено для профілю системи: «%s»"

#: ../server/rhnPackage.py:52 ../server/rhnPackage.py:98
#: ../server/rhnPackage.py:118 ../server/rhnPackage.py:179
#, python-format
msgid "Invalid RPM package %s requested"
msgstr "Надіслано запит щодо некоректного пакунка RPM %s"

#: ../server/rhnPackage.py:124
msgid "Package not found"
msgstr "Пакунок не знайдено"

#: ../server/rhnServer/server_class.py:103
#, python-format
msgid "Architecture `%s' is not supported"
msgstr "Підтримки архітектури «%s» не передбачено"

#: ../server/rhnServer/server_token.py:86
#, python-format
msgid "Token `%s' has more than one base channel assigned"
msgstr "Із ключем «%s» пов’язано декілька базових каналів"

#: ../server/rhnServer/server_token.py:89
msgid "Conflicting base channels"
msgstr "Конфлікт основних каналів"

#: ../server/rhnServer/server_token.py:223
#, python-format
msgid "Failed to add server to group %s"
msgstr "Не вдалося додати сервер до групи %s"

#: ../server/rhnServer/server_token.py:574
#, python-format
msgid "Maximum usage count of %s reached"
msgstr "Перевищено максимальну кількість використано %s"

#: ../server/rhnServer/server_token.py:869
msgid ""
"Stacking of re-registration tokens with different base entitlements is not "
"supported"
msgstr ""
"Підтримки стосування повторних реєстраційних ключів із різними базовими "
"прив’язками не передбачено"

#: ../server/rhnServer/server_token.py:946
#, python-format
msgid "Could not find token '%s'"
msgstr "Не вдалося знайти ключ «%s»"

#. Cannot use activation keys from different orgs
#: ../server/rhnServer/server_token.py:979
msgid "Tokens from mismatching orgs"
msgstr "Ключі з невідповідних установ"

#. Two tokens with different kickstart sessions
#: ../server/rhnServer/server_token.py:989
msgid "Kickstart session mismatch"
msgstr "Невідповідність сеансів kickstart"

#: ../server/rhnServer/server_token.py:1013
msgid "Stacking of re-registration tokens is not supported"
msgstr "Підтримки стосування повторних реєстраційних ключів не передбачено"

#: ../server/rhnUser.py:327 ../server/rhnUser.py:330
msgid "Invalid username/password combination"
msgstr "Неприпустима комбінація ім'я/пароль"

#. nope, not reserved either
#: ../server/rhnUser.py:527
#, python-format
msgid "Username `%s' has not been reserved"
msgstr "Ім’я користувача «%s» не було зарезервовано"

#. New accounts have to specify an e-mail address
#: ../server/rhnUser.py:533
msgid "E-mail address not specified"
msgstr "Не вказано адресу електронної пошти"

#: ../server/rhnUser.py:575
#, python-format
msgid "username should be at least %d characters"
msgstr "ім’я користувача має складатися принаймні з %d символів"

#: ../server/rhnUser.py:579
#, python-format
msgid "username should be less than %d characters"
msgstr "довжина імені користувача не повинна перевищувати %d символів"

#: ../server/rhnUser.py:593
#, python-format
msgid "username = `%s', invalid character `%s'"
msgstr "ім'я користувача = «%s», неприпустимий символ «%s»"

#: ../server/rhnUser.py:614
#, python-format
msgid "Please limit your e-mail address to %s chars"
msgstr ""
"Будь ласка, вкажіть адресу електронної пошти, що не перевищує %s символів у "
"довжину"

#: ../server/rhnUser.py:704
#, python-format
msgid "password must be at least %d characters"
msgstr "пароль має складатися принаймні з %d символів"

#: ../server/rhnUser.py:708
#, python-format
msgid "Password must be shorter than %d characters"
msgstr "Пароль має не перевищувати %d символів у довжину"

#: ../server/rhnUser.py:724
#, python-format
msgid "password contains character `%s'"
msgstr "пароль містить символ «%s»"

#: ../server/rhnUser.py:733
#, python-format
msgid "username should be at least %d characters long"
msgstr "ім’я користувача має складатися принаймні з %d символів"

#: ../server/rhnUser.py:744
#, python-format
msgid "Cannot register usernames ending with %s"
msgstr "Не можна реєструвати ім’я користувача, що завершується на %s"

#. 0-999: Red Hat client/client-like interaction errors:
#: ../common/rhnException.py:55
msgid "This does not appear to be a valid username."
msgstr "Здається, це не є коректним іменем користувача."

#: ../common/rhnException.py:56
msgid "Invalid username and password combination."
msgstr "Некоректне поєднання імені користувача і пароля."

#: ../common/rhnException.py:57
msgid "This login is already taken, or the password is incorrect."
msgstr "Це ім’я користувача вже використано або вказано неправильний пароль."

#: ../common/rhnException.py:58
msgid "Permission denied."
msgstr "Доступ заборонено."

#: ../common/rhnException.py:59
msgid "Object not found."
msgstr "Об’єкт не знайдено."

#: ../common/rhnException.py:60
msgid "Invalid System Digital ID."
msgstr "Некоректний цифровий ідентифікатор системи."

#: ../common/rhnException.py:61
msgid "Invalid System Credentials."
msgstr "Некоректні реєстраційні дані системи."

#: ../common/rhnException.py:62
msgid "Could not retrieve user data from database."
msgstr "Не вдалося отримати дані користувача з бази даних."

#: ../common/rhnException.py:63
msgid "Valid username required."
msgstr "Потрібне коректне ім’я користувача."

#: ../common/rhnException.py:64
msgid "Valid password required."
msgstr "Потрібен коректний пароль."

#: ../common/rhnException.py:65
msgid "Minimum username length violation."
msgstr "Порушено обмеження на мінімальну довжину імені користувача."

#: ../common/rhnException.py:66
msgid "Minimum password length violation."
msgstr "Порушення обмеження на мінімальну довжину пароля."

#: ../common/rhnException.py:67
msgid "The username contains invalid characters."
msgstr "У імені користувача містяться некоректні символи."

#: ../common/rhnException.py:68
msgid "File not found."
msgstr "Файл не знайдено."

#: ../common/rhnException.py:69
msgid "Architecture and OS version combination is not supported."
msgstr ""
"Підтримки такої комбінації архітектури і версії операційної системи не "
"передбачено."

#: ../common/rhnException.py:70
msgid "Could not retrieve system data from database."
msgstr "Не вдалося отримати дані щодо системи із бази даних."

#: ../common/rhnException.py:71
msgid "Invalid arguments passed to function."
msgstr "Функції передано некоректні аргументи."

#: ../common/rhnException.py:72
msgid "Unable to retrieve requested entry."
msgstr "Не вдалося отримати потрібний запис."

#: ../common/rhnException.py:73
msgid "Could not update database entry."
msgstr "Не вдалося оновити запис у базі даних."

#: ../common/rhnException.py:74
msgid "Unsupported server architecture."
msgstr "Непідтримувана архітектура сервера."

#: ../common/rhnException.py:76
#, fuzzy, python-format
msgid ""
"\n"
"     The anonymous server functionality is no longer available.\n"
"\n"
"     Please re-register this system by running mgr_register as root.\n"
"     Please visit https://%(hostname)s/rhn/systems/SystemEntitlements.do\n"
"     or login at https://%(hostname)s, and from the \"Overview\" tab,\n"
"     select \"Subscription Management\" to enable the service for this "
"system.\n"
"     "
msgstr ""
"\n"
"     Функціональні можливості анонімного сервера більше не доступні.\n"
"\n"
"     Будь ласка, повторно зареєструйте цю систему за допомогою команди "
"rhn_register,\n"
"     відданої від імені користувача root.\n"
"     Будь ласка, відвідайте https://%(hostname)s/rhn/systems/"
"SystemEntitlements.do\n"
"     або увійдіть до системи на сторінці https://%(hostname)s, а потім на "
"вкладці «Огляд» («Overview»),\n"
"     виберіть «Керування передплатами» («Subscription Management»), щоб "
"увімкнути службу для цієї системи.\n"
"     "

#: ../common/rhnException.py:85
msgid "Record not available in the database."
msgstr "Запис у базі даних є недоступним."

#: ../common/rhnException.py:86
msgid "Invalid value for entry."
msgstr "Некоректне значення для запису."

#: ../common/rhnException.py:88
#, fuzzy, python-format
msgid ""
"\n"
"     This system does not have a valid entitlement for SUSE Multi-Linux "
"Manager.\n"
"     Please visit https://%(hostname)s/rhn/systems/SystemEntitlements.do\n"
"     or login at https://%(hostname)s, and from the \"Overview\" tab,\n"
"     select \"Subscription Management\" to enable the service for this "
"system.\n"
"     "
msgstr ""
"\n"
"     Ця система не має коректної реєстрації у Red Hat Satellite.\n"
"     Будь ласка, відвідайте https://%(hostname)s/rhn/systems/"
"SystemEntitlements.do\n"
"     або увійдіть до системи на сторінці https://%(hostname)s, а потім на "
"вкладці «Огляд» («Overview»),\n"
"     виберіть «Керування передплатами» («Subscription Management»), щоб "
"увімкнути службу для цієї системи.\n"
"     \n"
"     "

#: ../common/rhnException.py:95
msgid "Channel error"
msgstr "Помилка каналу"

#: ../common/rhnException.py:96
msgid "Client session token is invalid."
msgstr "Ключ сеансу клієнта є некоректним."

#: ../common/rhnException.py:97
msgid "Client session token has expired."
msgstr "Строк дії ключа сеансу клієнта вичерпано."

#: ../common/rhnException.py:98
msgid "You are not authorized to retrieve the requested object."
msgstr "У вас немає прав для отримання запитаного об’єкта."

#: ../common/rhnException.py:99
msgid "Invalid action"
msgstr "Некоректна дія"

#: ../common/rhnException.py:101
msgid "You are not allowed to perform administrative tasks on this system."
msgstr "Ви не маєте права виконувати завдання з адміністрування у цій системі."

#: ../common/rhnException.py:104
msgid "The system is already subscribed to the specified channel."
msgstr "Систему вже підписано на вказаний канал."

#: ../common/rhnException.py:105
msgid "The system is not currently subscribed to the specified channel."
msgstr "Систему зараз не підписано на вказаний канал."

#: ../common/rhnException.py:106
msgid "The specified channel does not exist."
msgstr "Вказаного каналу не існує."

#: ../common/rhnException.py:107
msgid "Invalid channel version."
msgstr "Некоректна версія каналу."

#: ../common/rhnException.py:109
msgid ""
"\n"
"     User group membership limits exceeded.\n"
"\n"
"     The current settings for your account do not allow you to add another\n"
"     user account. Please check with the organization administrator for "
"your\n"
"     account if the maximum number of users allowed to subscribe to server "
"needs\n"
"     to be changed.\n"
"     "
msgstr ""
"\n"
"     Перевищено обмеження на участь у групі.\n"
"\n"
"     Поточні параметри для вашого облікового запису не дозволяють\n"
"     додавання ще одного облікового запису користувача. Будь ласка, "
"перевірте\n"
"     разом із адміністратором вашого облікового запису у вашій установі,\n"
"     чи слід змінювати дозволену максимальну кількість користувачів,\n"
"     які можуть передплачувати послуги сервера.\n"
"     "

#: ../common/rhnException.py:119
msgid ""
"\n"
"     System group membership limits exceeded.\n"
"\n"
"     The current settings for your account do not allow you to add another\n"
"     system profile. Please check with the organization administrator for "
"your\n"
"     account for modifying the maximum number of system profiles that can "
"be\n"
"     subscribed to your account.\n"
"     "
msgstr ""
"\n"
"     Перевищено обмеження на участь систем у групі.\n"
"\n"
"     Поточні параметри вашого облікового запису забороняють додавання\n"
"     ще одного профілю системи. Будь ласка, перевірте разом із "
"адміністратором\n"
"     вашого облікового запису в установі, чи слід змінити максимальну "
"кількість\n"
"     профілів систем, які можуть працювати у межах передплати вашого "
"облікового запису.\n"
"     "

#: ../common/rhnException.py:129
msgid ""
"\n"
"     Invalid architecture.\n"
"\n"
"     The architecture of the package is not supported by\n"
"     "
msgstr ""
"\n"
"     Некоректна архітектура.\n"
"\n"
"     Підтримки архітектури пакунка не передбачено у\n"
"     "

#: ../common/rhnException.py:136
msgid "Invalid RPM header"
msgstr "Некоректний заголовок RPM"

#: ../common/rhnException.py:138
msgid ""
"\n"
"    This system is already registered as a Salt Minion. If you want to "
"register it as a traditional client\n"
"    please delete it first via the web UI or API and then register it using "
"the traditional tools.\n"
"    "
msgstr ""

#. For the uploading tools
#: ../common/rhnException.py:144
msgid "Invalid information uploaded to the server"
msgstr "На сервер вивантажено некоректні дані"

#: ../common/rhnException.py:145
msgid "Error uploading network interfaces configuration."
msgstr "Помилка під час спроби вивантажити налаштування інтерфейсів мережі."

#: ../common/rhnException.py:147
msgid ""
"\n"
"     Package Upload Failed due to uniqueness constraint violation.\n"
"     Make sure the package does not have any duplicate dependencies or\n"
"     does not already exists on the server\n"
"     "
msgstr ""
"\n"
"     Спроба вивантаження пакунка зазнала невдачі через порушення умов "
"однозначності.\n"
"     Переконайтеся, що у пакунка немає подвійних залежностей і що\n"
"     пакунок вже не зберігається на сервері.\n"
"     "

#: ../common/rhnException.py:154
#, fuzzy
msgid ""
"\n"
"     The --force mgrpush option is disabled on this server.\n"
"     Please contact your SUSE Multi-Linux Manager administrator for more "
"help.\n"
"     "
msgstr ""
"\n"
"     На цьому сервері вимкнено можливість використання --force rhnpush. \n"
"     Будь ласка, зв’яжіться із вашим адміністратором Satellite, щоб отримати "
"додаткові відомості.\n"
"     "

#: ../common/rhnException.py:161
msgid ""
"\n"
"     The activation token specified could not be found on the server.\n"
"     Please retry with a valid key.\n"
"     "
msgstr ""
"\n"
"     Не вдалося знайти вказаний ключ активації на сервері.\n"
"     Будь ласка, повторіть спробу із коректним ключем.\n"
"     "

#: ../common/rhnException.py:166
msgid "Too many systems registered using this registration token"
msgstr ""
"За допомогою цього реєстраційного ключа зареєстровано надто багато систем"

#: ../common/rhnException.py:167
msgid "Token contains invalid, obsoleted or insufficient settings"
msgstr "У ключі містяться некоректні, застарілі або недостатні параметри"

#: ../common/rhnException.py:168
msgid "Conflicting activation tokens"
msgstr "Конфлікт ключів активації"

#: ../common/rhnException.py:171
msgid ""
"\n"
"     No matching base channel found for your system.\n"
"     "
msgstr ""
"\n"
"     Для вашої системи не знайдено відповідного базового каналу.\n"
"     "

#: ../common/rhnException.py:176
msgid ""
"\n"
"     You do not have subscription permission to the designated channel.\n"
"     Please refer to your organization's channel or organization\n"
"     administrators for further details.\n"
"     "
msgstr ""
"\n"
"     У вас немає права підписуватися на вказаний канал.\n"
"     Будь ласка, зверніться до опису каналу вашої організації або\n"
"     до адміністраторів організації, щоб дізнатися більше.\n"
"     "

#: ../common/rhnException.py:182
msgid "You can not unsubscribe from base channel."
msgstr "Не можна відписатися від базового каналу."

#: ../common/rhnException.py:183
#, fuzzy
msgid "SUSE Multi-Linux Manager or Proxy channel can not be subscribed."
msgstr "Не вдалося передплатити канал Satellite або Proxy."

#. 80-90: server group errors
#: ../common/rhnException.py:185
msgid "There was an error while trying to join the system to its groups"
msgstr "Під час спроби долучити систему до її груп сталася помилка"

#. 90-100: entitlement errors
#: ../common/rhnException.py:187
msgid "Unable to entitle system"
msgstr "Не вдалося прив’язати систему"

#. 100-109: e-mail and uuid related faults
#: ../common/rhnException.py:189
msgid "Maximum e-mail length violation."
msgstr "Перевищення обмеження на максимальний розмір поштового повідомлення."

#: ../common/rhnException.py:190
msgid "This system has been previously registered."
msgstr "Цю систему вже було зареєстровано."

#: ../common/rhnException.py:191
msgid "Invalid username"
msgstr "Некоректне ім’я користувача"

#. 140-159 applet errors
#: ../common/rhnException.py:193
msgid "Unable to look up server"
msgstr "Не вдалося знайти сервер"

#. 160-179: OSAD errors
#: ../common/rhnException.py:195
msgid "Required argument is missing"
msgstr "Пропущено обов’язковий аргумент"

#. 600-699: RHEL5+ EN errors
#: ../common/rhnException.py:197
msgid "No entitlement information tied to hardware"
msgstr "З обладнанням не пов’язано реєстраційних даних"

#: ../common/rhnException.py:198
msgid "Installation number is not entitling"
msgstr "Кількість встановлень не допускає до обслуговування"

#. 700-799: Additional user input verification errors.
#: ../common/rhnException.py:200
msgid "Maximum username length violation"
msgstr "Перевищено максимальну довжину імені користувача"

#: ../common/rhnException.py:201
msgid "Maximum password length violation"
msgstr "Перевищення максимальної довжини пароля"

#: ../common/rhnException.py:202
msgid "This user has read only API access. Action denied."
msgstr ""
"У цього користувача є доступ лише до читання за допомогою API. Дію "
"заборонено."

#: ../common/rhnException.py:203
msgid "System Name cannot be less than 1 character"
msgstr "Назва системи має складатися із принаймні 1 символу"

#. 1000-1999: Proxy specific errors:
#. issued by a Proxy to the client
#: ../common/rhnException.py:206
#, fuzzy
msgid "SUSE Multi-Linux Manager Proxy error."
msgstr "Помилка проксі-сервера Spacewalk."

#: ../common/rhnException.py:207
#, fuzzy
msgid "SUSE Multi-Linux Manager Proxy unable to login."
msgstr "Не вдалося увійти до системи проксі-сервера Spacewalk."

#: ../common/rhnException.py:210
#, fuzzy
msgid ""
"\n"
"     SUSE Multi-Linux Manager Proxy system ID does not match a Spacewalk "
"Proxy Server\n"
"     in the database.\n"
"     "
msgstr ""
"\n"
"     Ідентифікатор системи проксі Spacewalk не відповідає запису проксі-"
"сервера Spacewalk\n"
"     у базі даних.\n"
"     "

#: ../common/rhnException.py:215
#, fuzzy
msgid "SUSE Multi-Linux Manager Proxy session token is invalid."
msgstr "Некоректний ключ сеансу Proxy Spacewalk."

#: ../common/rhnException.py:216
#, fuzzy
msgid "SUSE Multi-Linux Manager Proxy session token has expired."
msgstr "Строк дії ключа сеансу Proxy Spacewalk вичерпано."

#: ../common/rhnException.py:220
#, fuzzy
msgid ""
"\n"
"      user creation is not allowed via mgr_register;\n"
"     please contact your sysadmin to have your account created.\n"
"     "
msgstr ""
"\n"
"     Створення записів користувачів за допомогою rhn_register заборонено;\n"
"     будь ласка, зв’яжіться із вашим системним адміністратором для створення "
"вашого облікового запису.\n"
"     "

#: ../common/rhnException.py:226
#, fuzzy
msgid ""
"\n"
"     This SUSE Multi-Linux Manager server is not allowed to use Inter Server "
"Sync on this server\n"
"     "
msgstr ""
"\n"
"     Цьому серверу satellite заборонено використання синхронізації Inter "
"Satellite на цьому satellite\n"
"     "

#: ../common/rhnException.py:231
#, fuzzy
msgid ""
"\n"
"     Inter Server Sync is disabled on this SUSE Multi-Linux Manager Server.\n"
"     "
msgstr ""
"\n"
"     Синхронізацію Inter Satellite вимкнено на цьому satellite.\n"
"     "

#. 3000-3999: XML dumper errors:
#: ../common/rhnException.py:236
msgid "Invalid datatype passed"
msgstr "Передано некоректний тип даних"

#: ../common/rhnException.py:237
msgid "Unable to retrieve channel"
msgstr "Не вдалося отримати канал"

#: ../common/rhnException.py:238
msgid "Unable to retrieve package"
msgstr "Не вдалося отримати пакунок"

#: ../common/rhnException.py:239
msgid "Unable to retrieve erratum"
msgstr "Не вдалося отримати список помилок"

#: ../common/rhnException.py:240
msgid "File is missing"
msgstr "Не вистачає файла"

#: ../common/rhnException.py:241
msgid "Function retrieval error"
msgstr "Помилка під час спроби отримання даних функції"

#: ../common/rhnException.py:242
msgid "Function execution error"
msgstr "Помилка під час спроби виконання функції"

#: ../common/rhnException.py:243
msgid "Missing version string"
msgstr "Не вистачає рядка версії"

#: ../common/rhnException.py:244
msgid "Invalid version string"
msgstr "Некоректний рядок версії"

#: ../common/rhnException.py:245
msgid "Mismatching versions"
msgstr "Невідповідність версій"

#: ../common/rhnException.py:246
msgid "Invalid channel version"
msgstr "Некоректна версія каналу"

#: ../common/rhnException.py:247
msgid "No comps file for channel"
msgstr "На каналі немає файла comps"

#: ../common/rhnException.py:248
msgid "Unable to retrieve comps file"
msgstr "Не вдалося отримати файл comps"

#. 4000 - 4999: config management errors
#: ../common/rhnException.py:250
msgid "Configuration action missing"
msgstr "Не вказано дії з налаштовування"

#: ../common/rhnException.py:251
msgid "File too large"
msgstr "Файл є надто великим"

#: ../common/rhnException.py:252
msgid "File contains binary data"
msgstr "У файлі містяться бінарні дані"

#: ../common/rhnException.py:253
msgid "Configuration channel is not empty"
msgstr "Канал налаштовування не є порожнім"

#: ../common/rhnException.py:254
msgid "Permission error"
msgstr "Помилка доступу"

#: ../common/rhnException.py:255
msgid "Content missing for configuration file"
msgstr "У файлі налаштувань не міститься даних"

#: ../common/rhnException.py:256
msgid "Template delimiters not specified"
msgstr "Не вказано роздільники шаблона"

#: ../common/rhnException.py:257
msgid "Configuration channel does not exist"
msgstr "Каналу налаштовування не існує"

#: ../common/rhnException.py:258
msgid "Configuration channel already exists"
msgstr "Канал налаштовування вже існує"

#: ../common/rhnException.py:259
msgid "File missing from configuration channel"
msgstr "На каналі налаштовування не вистачає файла"

#: ../common/rhnException.py:260
msgid "Different revision of this file is uploaded"
msgstr "Вивантажено іншу модифікацію цього файла"

#: ../common/rhnException.py:261
msgid "File already uploaded to configuration channel"
msgstr "Файл вже вивантажено до каналу налаштовування"

#: ../common/rhnException.py:262
msgid "File size exceeds remaining quota space"
msgstr "Розмір файла перевищує залишок квоти у сховищі даних"

#: ../common/rhnException.py:263
msgid "Full path of file must be specified"
msgstr "Шлях до файла має бути вказано повністю"

#: ../common/rhnException.py:264
msgid "Invalid revision number"
msgstr "Некоректний номер модифікації"

#: ../common/rhnException.py:265
msgid "Cannot compare files of different file type"
msgstr "Не можна порівнювати файли різних типів"

#. 5100 - 5199: scap results reporting error
#: ../common/rhnException.py:267
msgid "SCAP results file transfer is invalid or incomplete"
msgstr "Передавання файла результатів SCAP є некоректним або неповним"

#: ../common/rhnException.py:268
msgid "Error composing directory path for detailed SCAP results"
msgstr ""
"Помилка під час спроби утворення шляху до каталогу для докладних результатів "
"SCAP"

#: ../common/rhnException.py:269
msgid "Error composing file path for detailed SCAP results"
msgstr ""
"Помилка під час спроби утворення шляху до файла для докладних результатів "
"SCAP"

#: ../common/rhnException.py:270
msgid "Invalid content encoding"
msgstr "Некоректне кодування даних"

#: ../common/rhnException.py:317
#, fuzzy
msgid ""
"\n"
"     An error has occurred while processing your request. If this problem\n"
"     persists please enter a bug report at scc.suse.com.\n"
"     If you choose to submit the bug report, please be sure to include\n"
"     details of what you were trying to do when this error occurred and\n"
"     details on how to reproduce this problem.\n"
msgstr ""
"\n"
"     Під час спроби обробити ваш запит сталася помилка. Якщо цю\n"
"     проблему не вдасться вирішити, будь ласка, створіть звіт щодо вади на "
"bugzilla.redhat.com.\n"
"     Якщо ви вирішите створити такий звіт щодо вади, будь ласка, не забудьте "
"вказати у ньому\n"
"     подробиці щодо того, що саме ви робили, коли сталася помилка, а також "
"спосіб, у який\n"
"     цю проблему можна відтворити.\n"

#: ../common/rhnException.py:368
#, python-format
msgid ""
"Error Message:\n"
"    %s\n"
msgstr ""
"Повідомлення про помилку:\n"
"    %s\n"

#: ../common/rhnException.py:370
#, python-format
msgid "Error Class Code: %s\n"
msgstr "Код помилкового класу: %s\n"

#: ../common/rhnException.py:373
#, python-format
msgid "Error Class Info: %s\n"
msgstr "Дані помилкового класу: %s\n"

#: ../common/rhnException.py:375
#, python-format
msgid "Explanation: %s"
msgstr "Пояснення: %s"

#: ../common/rhnTB.py:220
#, python-format
msgid "ERROR: %s %s: %s"
msgstr "ПОМИЛКА: %s %s: %s"

#: ../common/rhnTB.py:221
#, python-format
msgid "TRACEBACK: %s"
msgstr "ТРАСУВАННЯ: %s"

#: ../satellite_tools/disk_dumper/iss.py:1634
#: ../satellite_tools/disk_dumper/iss.py:1640
#, python-format
msgid "format of %s should be at least YYYYMMDD.\n"
msgstr ""

#: ../satellite_tools/messages.py:28
#, python-format
msgid ""
"\n"
"ERROR: executing step %s. Error is:\n"
"%s\n"
msgstr ""

#: ../satellite_tools/messages.py:35
#, python-format
msgid ""
"\n"
"ERROR: there was a problem accessing the channel data from your temporary\n"
"       repository. Did you migrate all of the data from the channel ISOs\n"
"       do this directory? If so, please recheck the channels ISOs, ensure\n"
"       that you have them all, and then iteratively remount and repopulate\n"
"       the temporary repository (%s).\n"
msgstr ""

#: ../satellite_tools/messages.py:45
#, python-format
msgid ""
"\n"
"ERROR: there was a problem synchronizing the information.\n"
"       Error message: %s\n"
msgstr ""

#: ../satellite_tools/messages.py:52
#, python-format
msgid ""
"\n"
"ERROR: There was a problem communicating with the ISS Master.\n"
"       Depending on the specific error details, please review your "
"configuration,\n"
"       basic network connectivity, and/or name resolution, and try again.\n"
"       Error message: %s\n"
msgstr ""

#: ../satellite_tools/messages.py:61
#, python-format
msgid ""
"\n"
"ERROR: a child-channel cannot be synced without its parent being synced as\n"
"       well. A parent needs to be either (a) previously synced or (b) "
"synced\n"
"       in tandem with the desired child-channel. Missing parents for this\n"
"       transaction:\n"
"       %s\n"
msgstr ""

#: ../satellite_tools/messages.py:71
#, python-format
msgid ""
"\n"
"ERROR: you are not entitled to sync a channel in this set of channels.\n"
"Please contact your SUSE Multi-Linux Manager contact\n"
"%s"
msgstr ""

#: ../satellite_tools/messages.py:77
#, python-format
msgid "  ERROR: not enough free space (%s KB) on device."
msgstr ""

#: ../satellite_tools/messages.py:79
#, python-format
msgid "    %3d/%s Fetch successful: %s (%s bytes)"
msgstr ""

#: ../satellite_tools/messages.py:80
#, fuzzy, python-format
msgid "    Extinct package:  %s"
msgstr "Мало бути вказано пакунок, а не: %s"

#: ../satellite_tools/messages.py:81
#, python-format
msgid "   Total size: %s"
msgstr ""

#: ../satellite_tools/messages.py:83
#, python-format
msgid "      Downloaded %s of %s. Estimated remaining time: %s"
msgstr ""

#: ../satellite_tools/messages.py:85
#, python-format
msgid "    Fetch unsuccessful: %s"
msgstr ""

#: ../satellite_tools/messages.py:86
#, python-format
msgid "   RPM fetch summary: %s"
msgstr ""

#: ../satellite_tools/messages.py:87
#, python-format
msgid "       success: %d"
msgstr ""

#: ../satellite_tools/messages.py:88
#, python-format
msgid "       failed:  %d"
msgstr ""

#: ../satellite_tools/messages.py:89
#, python-format
msgid "       extinct: %d"
msgstr ""

#: ../satellite_tools/messages.py:91
#, python-format
msgid "   Retrieving / parsing *relevant* package metadata: %s (%s)"
msgstr ""

#: ../satellite_tools/messages.py:92
#, python-format
msgid "   Retrieving / parsing errata data: %s (%s)"
msgstr ""

#: ../satellite_tools/messages.py:93
#, python-format
msgid "   Retrieving / parsing kickstart data: %s (%s)"
msgstr ""

#: ../satellite_tools/messages.py:94
#, python-format
msgid "   Retrieving / parsing kickstart tree files: %s (%s)"
msgstr ""

#: ../satellite_tools/messages.py:95
#, python-format
msgid "   Importing *relevant* package metadata: %s (%s)"
msgstr ""

#: ../satellite_tools/messages.py:96
msgid "   * WARNING: this may be a slow process."
msgstr ""

#: ../satellite_tools/messages.py:97
#, fuzzy
msgid "Linking packages to channels"
msgstr "Конфлікт основних каналів"

#: ../satellite_tools/messages.py:98
#, python-format
msgid "   Importing *relevant* errata: %s (%s)"
msgstr ""

#: ../satellite_tools/messages.py:99
msgid "   No new kickstartable tree to import"
msgstr ""

#: ../satellite_tools/messages.py:100
#, python-format
msgid "Importing kickstartable trees (%d)"
msgstr ""

#: ../satellite_tools/messages.py:101
#, python-format
msgid "Imported kickstartable trees (%d)"
msgstr ""

#: ../satellite_tools/reposync.py:138
msgid "+++ sending log as an email +++"
msgstr ""

#: ../satellite_tools/reposync.py:141
#, python-format
msgid "%s sync. report from %s"
msgstr ""

#: ../satellite_tools/reposync.py:151
msgid "+++ email requested, but there is nothing to send +++"
msgstr ""

#, fuzzy
#~ msgid "ISS is disabled on this server."
#~ msgstr "ISS вимкнено для цього satellite."

#, python-format
#~ msgid "Server \"%s\" is not enabled for ISS."
#~ msgstr "Сервер «%s» не увімкнено для ISS."

#, fuzzy
#~ msgid "base-channels"
#~ msgstr "Конфлікт основних каналів"

#, fuzzy, python-format
#~ msgid "Unable to save file %s: %s"
#~ msgstr "Не вдалося прив’язати систему"

#, fuzzy
#~ msgid "print the configuration and exit"
#~ msgstr "Каналу налаштовування не існує"

#, fuzzy, python-format
#~ msgid "ERROR: Unable to lookup Org Id %s"
#~ msgstr "Не вдалося знайти сервер"

#, fuzzy
#~ msgid "  12 - Channel do not exist"
#~ msgstr "Вказаного каналу не існує."

#, fuzzy, python-brace-format
#~ msgid "Your system was not found in the {PRODUCT_NAME} database"
#~ msgstr "Вашої системи не знайдено у базі даних RHN"

#, python-format
#~ msgid "Expected a package name, not: %s"
#~ msgstr "Мало бути вказано назву пакунка, а не: %s"

#, python-format
#~ msgid "Expected a package, not: %s"
#~ msgstr "Мало бути вказано пакунок, а не: %s"

#, python-format
#~ msgid "Invalid action value type %s (%s)"
#~ msgstr "Неприпустиме значення дії типу %s (%s)"

#, python-format
#~ msgid "Action %s does not belong to server %s"
#~ msgstr "Дія %s не належить до сервера %s"

#~ msgid "Required data missing"
#~ msgstr "Не вистачає потрібних даних"

#~ msgid "Required members missing"
#~ msgstr "Не вистачає потрібних елементів"

#~ msgid "Attempt to register a system to an invalid username"
#~ msgstr "Спроба зареєструвати систему на некоректне ім’я користувача"

#~ msgid "Invalid system release version requested"
#~ msgstr "Надіслано запит щодо некоректної версії випуску системи"

#, fuzzy
#~ msgid "Privacy Statement"
#~ msgstr "Угода щодо конфіденційності Red Hat Satellite"

#~ msgid "Expected a dictionary as a product argument"
#~ msgstr "Мало бути вказано словник як аргумент продукту"

#, python-format
#~ msgid "Invalid value '%s' for %s (%s)"
#~ msgstr "Некоректне значення «%s» для %s (%s)"

#~ msgid "Invalid system_id/action_id pair."
#~ msgstr "Некоректна пара ідентифікатор системи/ідентифікатор дії."

#, python-format
#~ msgid "Invalid value %s (%s)"
#~ msgstr "Некоректне значення %s (%s)"

#~ msgid "Crash information is invalid or incomplete"
#~ msgstr "Дані щодо аварії є некоректними або неповними"

#~ msgid "Crash file information is invalid or incomplete"
#~ msgstr "Дані з файла даних аварії є некоректними або неповними"

#~ msgid "Error composing crash directory path"
#~ msgstr "Помилка під час спроби утворення шляху до каталогу даних аварії"

#~ msgid "Error composing crash file path"
#~ msgstr "Помилка під час спроби утворення шляху до файла даних аварії"

#~ msgid "Invalid crash name"
#~ msgstr "Некоректна назва аварії"

#~ msgid "Crash reporting is disabled for this organization"
#~ msgstr "Для цієї організації звітування про вади вимкнено"

#~ msgid ""
#~ "\n"
#~ "If you have a registration number, please register with it first at "
#~ "http://www.redhat.com/apps/activate/ and then try again.\n"
#~ "\n"
#~ msgstr ""
#~ "\n"
#~ "Якщо у вас є реєстраційний номер, будь ласка, спершу зареєструйте його на "
#~ "http://www.redhat.com/apps/activate/, а потім повторіть спробу.\n"
#~ "\n"

#~ msgid ""
#~ "\n"
#~ "Invalid Architecture and OS release combination (%s, %s).\n"
#~ "Your system has been registered, but will not receive updates\n"
#~ "because it is not subscribed to a channel. If you have not yet\n"
#~ "activated your product for service, please visit our website at:\n"
#~ "\n"
#~ "     http://www.redhat.com/apps/activate/\n"
#~ "\n"
#~ "...to activate your product."
#~ msgstr ""
#~ "\n"
#~ "Некоректна комбінація архітектури і випуску операційної системи (%s, "
#~ "%s).\n"
#~ "Вашу систему зареєстровано, але вона не отримуватиме оновлень, оскільки\n"
#~ "її не підписано на канал. Якщо ви ще не активували ваш продукт\n"
#~ "для обслуговування, будь ласка, відвідайте наш сайт за адресою:\n"
#~ "\n"
#~ "     http://www.redhat.com/apps/activate/\n"
#~ "\n"
#~ "...щоб активувати ваш продукт."

#~ msgid "Red Hat Satellite Welcome Message"
#~ msgstr "Вітальне повідомлення Red Hat Satellite"
